package.json

```diff
@@ -48,10 +60,14 @@
     "jest-fetch-mock": "^3.0.3",
     "prettier": "^2.3.2",
     "react": "^17.0.2",
-    "ts-node": "^9.1.1",
+    "ts-node": "^10.8.1",
     "typescript": "^4.3.5"
   },
   "dependencies": {
-    "remix-auth": "^3.0.0"
+    "ethers": "^5.7.0",
+    "express": "^4.18.1",
+    "keccak": "^3.0.2",
+    "remix-auth": "^3.0.0",
+    "siwe": "^2.0.5"
```

Comment:
I think we can move `ethers` and, possibly, `siwe` to `peerDependencies` — this will allow the users to choose/upgrade package versions directly, without duplicating installs. A good way to ensure things don't break is to use a lower/upper supported version range, e.g., `"ethers": ">= 5.6.8 <6.0.0"` (ethers has a security vulnerability fixed in 5.6.8).